packages/cfd/src/features/Containers/top-up-demo-modal.tsx

```diff
@@ -29,9 +42,39 @@ const TopUpDemoModal = observer(({ platform }: TTopUpDemoModalProps) => {
         closeSuccessTopUpModal,
     } = ui;
 
-    const { current_account, dxtrade_companies, mt5_companies, topUpVirtual } = useCfdStore();
+    const { current_account } = useCfdStore();
+    const invalidate = useInvalidateQuery();
+    const { mutateAsync: MT5Deposit, status: mt5_deposit_status, error: mt5_deposit_error } = useMT5Deposit();
+    const {
+        mutateAsync: OtherCFDPlatformsDeposit,
+        status: other_cfd_deposit_status,
+        error: other_cfd_deposit_error,
+    } = useOtherCFDPlatformsDeposit();
 
     const ctrader_companies = getCTraderCompanies();
+    const dxtrade_companies = getDxCompanies();
+    const mt5_companies = getMtCompanies(is_eu);
+
+    useEffect(() => {
+        if (mt5_deposit_status === QUERY_STATUS.SUCCESS) invalidate('mt5_login_list');
+    }, [mt5_deposit_status, mt5_deposit_error]);
+
+    useEffect(() => {
+        if (other_cfd_deposit_status === QUERY_STATUS.SUCCESS) invalidate('trading_platform_accounts');
```

Comment:
same here

Reply:
Update the hook file like this

<img width="890" alt="image" src="https://github.com/binary-com/deriv-app/assets/120543468/cea52771-9e81-4b56-9309-73e40e06e6aa">